tools/monorepo-utils/src/index.ts

```diff
@@ -9,12 +9,39 @@ import chalk from 'chalk';
  * Internal dependencies
  */
 import CodeFreeze from './code-freeze/commands';
+import Slack from './slack/commands/slack';
+import { Logger } from './core/logger';
+import { isGithubCI } from './core/environment';
 
-console.log(
-	chalk.rgb( 150, 88, 138 ).bold( figlet.textSync( 'WooCommerce Utilities' ) )
-);
+if ( ! isGithubCI() ) {
+	Logger.notice(
+		chalk
+			.rgb( 150, 88, 138 )
+			.bold( figlet.textSync( 'WooCommerce \n Utils' ) )
```

Comment:
Don't bother logging this in CI and also split with newline, because it looks better on smaller screens I think